cmd/prowjob-controller/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"time"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api"
+	"k8s.io/klog/v2"
+	clientset "k8s.io/test-infra/prow/client/clientset/versioned"
+	informers "k8s.io/test-infra/prow/client/informers/externalversions"
+
+	controller "github.com/openshift/release-controller/pkg/controller/prowjob"
+	"github.com/openshift/release-controller/pkg/signals"
+)
+
+type options struct {
+	clusterContext string
+	authInfo       string
+	prowNamespace  string
+}
+
+func main() {
```

Comment:
Instead of having one binary for each controller, how about having a single binary that runs the related set of controllers.  One example is https://github.com/openshift/cluster-kube-controller-manager-operator/blob/855d161acbd6a3e80938b8ffcf5688e68dd962a9/pkg/operator/starter.go#L210-L222.  That `Run` function takes the context input and produces several different controllers which are started in a single binary.